core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/SeekableByteArrayInputStream.java

```diff
@@ -115,14 +114,14 @@ public void reset() {
   public void close() throws IOException {}
 
   public SeekableByteArrayInputStream(byte[] buf) {
-    Preconditions.checkNotNull(buf, "bug argument was null");
+    Objects.requireNonNull(buf, "bug argument was null");
     this.buffer = buf;
     this.cur = 0;
     this.max = buf.length;
   }
 
   public SeekableByteArrayInputStream(byte[] buf, int maxOffset) {
-    Preconditions.checkNotNull(buf, "bug argument was null");
+    Objects.requireNonNull(buf, "bug argument was null");
     this.buffer = buf;
```

Comment:
These are more readable if they are inline'd into the assignment. I'm not sure if this worked for Guava Preconditions, but the Java requireNonNull returns the reference, so it does work for that. Example:

```java
    this.buffer = Objects.requireNonNull(buf, "buf argument was null");
```

But, we should stick with whatever minimal change merges to master easiest.

Reply:
I changed it to what we have in master, which is static import on two lines.